composer.json

```diff
@@ -28,11 +28,12 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^9.3",
-        "squizlabs/php_codesniffer": "^3.6"
+        "squizlabs/php_codesniffer": "^3.6",
```

Comment:
Note for @christyjacob4 , can we now remove this as we are not using phpcbf/phpcs anymore?